src/main/java/com/minelittlepony/settings/Value.java

```diff
@@ -0,0 +1,19 @@
+package com.minelittlepony.settings;
+
+import com.minelittlepony.gui.IGuiCallback;
+
+public interface Value<T> extends IGuiCallback<T> {
+
+    T get();
+
+    void set(T value);
+
+    @Override
+    default void perform(T in) {
```

Comment:
Can't we just use `value::set` whenever we need to use this as a callback?

Reply:
Because like this we can just pass value in as it is?